test/integration/instance.validations.test.ts

```diff
@@ -0,0 +1,978 @@
+'use strict';
+
+import type {
+  InferAttributes,
+  CreationOptional,
+  InferCreationAttributes,
+  HasManyAddAssociationMixin,
+  BelongsToSetAssociationMixin,
+  HasMany,
+  BelongsTo,
+  Utils,
+} from '@sequelize/core';
+import {
+  DataTypes,
+  ValidationError,
+  Model,
+  Sequelize,
+} from '@sequelize/core';
+import chai from 'chai';
+import {
+  getTestDialectTeaser,
+  getTestDialect,
+  sequelize,
+  beforeEach2,
+} from './support';
+
+const expect = chai.expect;
+
+describe(getTestDialectTeaser('InstanceValidator'), async () => {
+  describe('#update', async () => {
+    it('should allow us to update specific columns without tripping the validations', async () => {
+      class User extends Model<InferAttributes<User>, InferCreationAttributes<User>> {
+        declare id?: CreationOptional<number>;
```

Comment:
The ID is already `CreationOptional`, you don't need to make it fully optional (same for all tests)

```suggestion
        declare id: CreationOptional<number>;
```